src/runs/execution/execute.ts

```diff
@@ -144,6 +145,7 @@ export async function executeRun(run: LoadedRun) {
       );
 
       await agentRun;
+      if (isOpenTelemetryEnabled) await batchSpanProcessor.forceFlush();
```

Comment:
Is this necessary? I got the impression that opentelemetry should quietly work outside of your code and flush regularly on its own